include/mesos/v1/mesos.proto

```diff
@@ -1783,6 +1783,12 @@ message ResourceStatistics {
   // Total memory + swap usage. This is set if swap is enabled.
   optional uint64 mem_total_memsw_bytes = 37;
 
+  // Current kernel memory allocation
+  optional uint64 mem_kmem_usage_bytes = 52;
```

Comment:
Is there a reason the valued don't match `include/mesos/mesos.proto`?

Reply:
There isn't a good reason. I've updated `mesos/mesos.proto` to match `mesos/v1/mesos.proto` (i.e. use the smaller field numbers).